docs/source/optimizations/eeroq/eeroq_css.ipynb

```diff
@@ -18,18 +18,20 @@
   },
```

Comment:
<p>This probably doesn't need to be a subheading.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/1126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Yeah, it looked weird to me too in ReviewNB, but it's not set as a subheading afaict: 
![image](https://github.com/user-attachments/assets/10cfd14c-d821-4054-af6a-10a8bd442326)

Will correct in a follow-up PR if its wrong on Read-the-docs too